src/open-id-connect-configuration-manager.ts

```diff
@@ -47,11 +47,11 @@ export default class {
         return this._userManagerSettings;
     }
 
-    public get redirectUri(): string {
+    public get redirectUri(): string | undefined {
```

Comment:
Hmm. This is an interesting one for me. Why was the union with `undefined` necessary?

Reply:
I am using the strict mode of typescript, that warns about possible problems.
the ```get redirectUri()``` returns a type ```uri?: string;```.
Necessary? no depend on your 'tsconfig.json' settings.
I did changes in more places that need to be evaluated. like the bang operator.
```this.openIdConnectConfiguration.redirectUri``` TS warns for the possibility of undefined.
```this.openIdConnectConfiguration.redirectUri!``` is a way of saying I have seen/checked it and accept it.

Reply:
Thank you for letting me know. Increasing the stringency of the TypeScript compiler will improve the code.